app/boards/shields/sofle_rgb_keyhive/sofle_rgb_keyhive.conf

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022, Josh Whitley
+# SPDX-License-Identifier: MIT
+
+# Uncomment the following line to enable the Sofle OLED Display
+# CONFIG_ZMK_DISPLAY=y
+
+# Uncomment the following two lines to disable the underglow LEDs
+# CONFIG_ZMK_RGB_UNDERGLOW=n
+# CONFIG_WS2812_STRIP=n
+
+CONFIG_EC11=y
+CONFIG_EC11_TRIGGER_GLOBAL_THREAD=y
```

Comment:
Does every sofle rgb have encoders?

Reply:
Im not OP, but have an RGB Sofle from Keyhive.
Every kit comes with encoders. One for each half.

Reply:
Yes, this is correct. It has encoders on each half which are not optional.